tutorial/transport/py_transport.ipynb

```diff
@@ -383,7 +365,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scen.solve(model='dantzig')"
+    "from tutorial.transport.linopy_model import create_dantzig_model, store_dantzig_solution\n",
+    "\n",
+    "m = create_dantzig_model(i=i, j=j, a=a, b=b, d=d, f=f)\n",
```

Comment:
The Dantzig model code knows which sets and parameters are needed so this can all be read from the `run` directly...
```suggestion
    "m = create_dantzig_model(run)\n",
```

Reply:
I thought about that and I'm not sure I agree. Passing only a `Run` means we have to document the requirements for this `Run`: it needs to have these `IndexSet`s, `Parameter`s, etc. The way it is now, we only explicitly pass in the data we need, or maybe even still too much. In practice, it might suffice to just pass in `i.elements` and `i.name` etc.

Reply:
But  think about the signature for creating a message_ix model, which has dozens of sets and parameters. This will become extremely painful.

In practice, the "model" knows which sets and parameters are required, so if a **Run** is passed that doesn't have a required item, the model (i.e., the model-creation-function) will just raise an error.

Reply:
Theoretically, the signature for something like `create_message_model()` could be different from `create_dantzig_model()`, but fine, I'll just pass a run.

Reply:
Please note, though, that passing a run requires additional DB calls to `get()` the indexsets etc, which likely slows down runtime. I doubt that it will notably affect the transport tutorial, though.